core/src/main/java/org/apache/metamodel/UpdateSummaryBuilder.java

```diff
@@ -1,4 +1,5 @@
 /**
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Surprised me as well, initally i put this check in place only for testing purposes, but differences in multiple locations were found, i commited it.
As checkstyle does a 1:1 check, also the /** vs /* issue raised in membrane can be centrally defined here.